examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md

```diff
@@ -0,0 +1,41 @@
+# quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration
+
+
+
+## Overview
+
+![simple-quickstart-arch](https://user-images.githubusercontent.com/5663078/229119386-0dbc6e30-a060-465e-86dd-f89712b0fc49.png)
+
+This Partner Solutions template enables you to begin working with your Machine Learning models rapidly, using your MongoDB Atlas Cluster and Amazon SageMaker Endpoint as tools. With this template, you can utilize MongoDB as a data source and Amazon SageMaker for data analysis, streamlining the process of building and deploying machine learning models.
+
+
+
+## MongoDB Atlas CFN Resources used by the templates
+
+- [MongoDB::Atlas::Trigger](../../cfn-resources/trigger/)
+
+
+## Environment Configured by the Partner Solutions template
+The Partner Solutions template will generate the following resources:
+ - Configure [MongoDB Partner Event Bus](http://mongodb.com/docs/atlas/app-services/triggers/aws-eventbridge/#std-label-aws-eventbridge).
+ - Set up a [database trigger](https://www.mongodb.com/docs/atlas/app-services/triggers/database-triggers/) with your Atlas Cluster.
+ - Create lambda functions to run the machine learning model and send the classification results to your MongoDB Atlas Cluster. 
+    - See [iris_classifier](sagemaker-example/iris_classifier/) for an example of machine learning model to use with this template.
+    -  See [lambda_functions](sagemaker-example/lambda_functions/) for an example of lambda functions to use to read and write data to your MongoDB Atlas cluster.
+
+
+## Permissions required to run the template
+See [execution-role.yaml](execution-role.yaml) for the list of permissions needed to run the template. 
+
+
+If you want to create an IAM (Identity and Access Management) role to run the quickstart template on a CFN (CloudFormation) stack, you can use the [execution-role.yaml](execution-role.yaml) file. Note that passing an IAM role to CloudFormation when creating a stack is optional. If you don't supply one, the user permissions are assumed. 
```

Comment:
What are the default permissions? What does "user permissions" refer to?

Reply:
> What does "user permissions" refer to?

When you log in to the AWS console, you must provide credentials for your AWS user. User permissions are those associated with your AWS user that you used to log in to the AWS console.

> What are the default permissions?

It depends on how you set up your AWS account.